package.json

```diff
@@ -3,7 +3,7 @@
   "name": "ambire-common",
   "description": "Common ground for the Ambire apps",
   "scripts": {
-    "generate:humanizerSelectorsAndNames": "node scripts/compileContractSelectorsAndNames.js",
+    "humanizer:addMMList": "node scripts/addMetamaskContractMapToHumanizerConsts.js",
```

Comment:
Minor, but that's too generic. The file name is also a bit confusing. Since this script is quite specific, I'd add a quite specific name here too like: `updateHumanizerInfoWithMetaMaskContactMetadata.js` and the script name could be `humanizer:update:metamaskContactMetadata` or something similar.

If you want to introduce `humanizer:` prefix to all related scripts, that's fine, but also update all others.